File: clients/android/NewsBlur/app/src/main/java/com/newsblur/widget/WidgetRemoteViews.java
package com.newsblur.widget;

import androidx.annotation.ColorInt;
import android.widget.RemoteViews;

class WidgetRemoteViews extends RemoteViews {

    WidgetRemoteViews(String packageName, int layoutId) {
        super(packageName, layoutId);
    }

    void setViewBackgroundColor(int viewId, @ColorInt int color) {
        setInt(viewId, "setBackgroundColor", color);
    }
}

File: clients/android/NewsBlur/app/src/main/java/com/newsblur/widget/WidgetRemoteViews.kt
package com.newsblur.widget

import android.widget.RemoteViews
import androidx.annotation.ColorInt

internal class WidgetRemoteViews(
        packageName: String,
        layoutId: Int,
) : RemoteViews(packageName, layoutId) {

    fun setViewBackgroundColor(viewId: Int, @ColorInt color: Int) {
        setInt(viewId, "setBackgroundColor", color)
    }
}